api/src/api/opportunities/opportunity_schemas.py

```diff
@@ -74,3 +79,26 @@ class OpportunitySearchSchema(request_schema.OrderedSchema):
         required=True,
     )
     paging = fields.Nested(PaginationSchema(), required=True)
+
+
+class OpportunitySearchHeaderSchema(request_schema.OrderedSchema):
```

Comment:
This is outside the scope of this ticket, but when reviewing this, I'm noting we could [remove the OrderedSchema workaround](https://github.com/navapbc/template-application-flask/issues/59#issuecomment-1792425510) in a future ticket.

Reply:
When we do the restructuring for error formatting of responses, I'll do it then. That approach will need a new base schema that we derive everything from to make some of the error formatting utils work happily.